src/arrays.js

```diff
@@ -1,61 +1,82 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index >= array.length) 
+  return array[index - array.length];
+  return array[index]
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.toString()
 };
 
-const csvStringToArray = string => {
-  // your code here
-};
+function csvStringToArray(string) {
+  return string.split(",", 5)
+}
 
-const addToArray = (element, array) => {
-  // your code here
+const addToArray = (elementX, arrayX) => {
+  arrayX.push(elementX)
 };
 
-const addToArray2 = (element, array) => {
-  // your code here
+const addToArray2 = (NEWelement, ORIGarray) => {
+  let addToArray3;
+  addToArray3 = ORIGarray.concat(NEWelement)
+  return addToArray3
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+ return array.splice(index,1)
 };
 
+
 const numbersToStrings = numbers => {
-  // your code here
+  return numbers.map(String)
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  return strings.map(function(A){return A.toUpperCase();})
 };
 
 const reverseWordsInArray = strings => {
-  // your code here
+   return strings.map(function(A){return A.split("").reverse().join("");})
 };
 
 const onlyEven = numbers => {
-  // your code here
+//const numbers2 = numbers.filter(a => a % 2 === 0); works, replaced filter with map,but this gives T or F//
+return numbers.filter(a => a % 2 === 0);
 };
 
+//splice() method mutates an array and returns the removed items. Splice example with no mutation.
 const removeNthElement2 = (index, array) => {
-  // your code here
+let newArray = [...array]
+if (index >= 0) {
+    newArray.splice(index, 1)
+    return newArray
+  }
 };
 
 const elementsStartingWithAVowel = strings => {
-  // your code here
+  let results = []
+  for (let i = 0; i <strings.length; i++) {
+    if ((strings[i].startsWith('a')) || (strings[i].startsWith('e')) || (strings[i].startsWith('i'))  
+    || (strings[i].startsWith('o')) || (strings[i].startsWith('u')) || (strings[i].startsWith('A')) 
+    || (strings[i].startsWith('E')) || (strings[i].startsWith('I')) || (strings[i].startsWith('O')) 
+    || (strings[i].startsWith('U')))
+    results.push(strings[i])
+    }
+    return results
```

Comment:
The basic idea is here, but theres a few different ways to go about this. This has a lot of repetition, and isnt that maintainable so should be re-written. One way is to have an array of values to check for, then use a combination of the filter and includes array methods, and the toLowerCase string method:
```
  const vowels = ['a', 'e', 'i', 'o', 'u'];

  return strings.filter(string => vowels.includes(string[0].toLowerCase()));
```
Much more concise and does the exact same job